code/modules/reagents/chemistry/reagents/water.dm

```diff
@@ -112,7 +127,13 @@
 
 /datum/reagent/blood/on_merge(list/mix_data)
 	if(data && mix_data)
-		data["cloneable"] = 0 //On mix, consider the genetic sampling unviable for pod cloning, or else we won't know who's even getting cloned, etc
+		data["cloneable"] = 0 //On mix, consider the genetic sampling unviable for pod cloning, or else we won't know who's even getting cloned, etc coagulated
+		if(data["species_only"] == TRUE && data["species"] != mix_data["species"] || mix_data["species_only"] == TRUE && data["species"] != mix_data["species"] == "Vox") // Vox blood with non vox blood
```

Comment:
I think the first part of this or check is all we need?
`mix_data["species_only"] == TRUE && data["species"] != mix_data["species"] == "Vox"` this check isnt needed
and its- compairing 3 things? 
test the PR without this part and I think it will still work.

Reply:
== "Vox" Was removed, rest of that check was needed, details in pr_review